onitu/plug/router.py

```diff
@@ -32,7 +37,9 @@ def __init__(self, plug):
 
         self.handlers = {
             CHUNK: self._handle_get_chunk,
-            FILE: self._handle_get_file
+            FILE: self._handle_get_file,
+            GET_CHUNK: self._handle_get_chunk,
+            GET_FILE: self._handle_get_file
```

Comment:
Because I don't want to change the FILE and CHUNK everywhere. You already did it in the oauth branch and it will be merged. If I do it in this branch, I will just duplicate the work.